src/pages/wheelcalc.tsx

```diff
@@ -0,0 +1,165 @@
+  numDays: number
+  wheelCredit: boolean
+  runecube: boolean
+}
+
+const DEFAULT_DATA: WheelData = {
+  numSpins: 500,
+  numDays: 7,
+  wheelCredit: true,
+  runecube: true,
+}
+
+export default () => {
+  const [data, values, setValues] = Calculator.useData(DEFAULT_DATA, (settings) => ({
+    numSpins: settings.numSpins ? parseInt(settings.numSpins, 10) : undefined,
+    numDays: settings.numDays ? parseInt(settings.numDays, 10) : undefined,
+    wheelCredit: settings.wheelCredit === undefined ? undefined : !!settings.wheelCredit,
+    runecube: settings.runecube === undefined ? undefined : !!settings.runecube,
+  }))
+
+  // Calculations.
+  const wheelCost = data.wheelCredit ? 5 : 10;
+
+  let coinsTotal = 0;
+  let spinsPerDayString = "";
+  let coinsPerDayString = "";
+		
+  if (data.numSpins % data.numDays !== 0) {
+    let spinsPerDay = Math.floor(data.numSpins / data.numDays);
+    let extraSpinDays = data.numSpins % data.numDays;
+    let regularDays = data.numDays - extraSpinDays;
+    spinsPerDayString =  (spinsPerDay + 1).toLocaleString() + " spins per day for " + extraSpinDays.toLocaleString() + " days, and " +  spinsPerDay.toLocaleString() + " for " + regularDays.toLocaleString() + " days";
+
+    let coinsPerDay = 0;
+    let coinsPerExtraSpinDay = 0;
+    for (let i = 0; i < (spinsPerDay + 1); i++)
+    {
+        coinsPerExtraSpinDay += i*wheelCost;
+    }
+    for (let i = 0; i < spinsPerDay; i++)
+    {
+        coinsPerDay += i * wheelCost;
+    }
+
+    coinsPerDayString = coinsPerExtraSpinDay.toLocaleString() + " coins per day for " + extraSpinDays.toLocaleString() + " days, and " + coinsPerDay.toLocaleString() + " for " + regularDays.toLocaleString() + " days";
+    coinsTotal = coinsPerDay * regularDays + coinsPerExtraSpinDay * extraSpinDays;
+  } else {
+    let spinsPerDay = data.numSpins / data.numDays;
+    spinsPerDayString =  spinsPerDay.toLocaleString() + " spins per day";
+
+    let coinsPerDay = 0;
+    for (let i = 0; i < spinsPerDay; i++)
+    {
+        coinsPerDay += i * wheelCost;
+    }
+    coinsPerDayString = coinsPerDay.toLocaleString() + " coins per day";
+    coinsTotal = coinsPerDay * data.numDays;
+  }
+
+  const chestsWithoutExploring = Math.ceil(coinsTotal / 100);
+  
+  let coinsPerOJ = 0.0;
+  let coinsPerCider = 0.0;
+  let coinsPerLemonade = 0.0;
+  let coinsPerAP = 0.0;
+    
+  let chestsPerOJ = 0.0;
+  let chestsPerCider = 0.0;
+  let chestsPerLemonade = 0.0;
+  let chestsPerAP = 0.0;
+  
+  if (data.runecube) {	
+    coinsPerOJ = 1/1.1;
+    coinsPerCider = 9.11;
+    coinsPerLemonade = 1/2.74;
+    coinsPerAP = 9.11;
+
+    chestsPerOJ = 1/65.91;
+    chestsPerCider = 1/6.56;
+    chestsPerLemonade = 1/163.96;
+    chestsPerAP = 1/6.56;
+  } else {	
+    coinsPerOJ = 1/1.27;
+    coinsPerCider = 7.89;
+    coinsPerLemonade = 1/4.56;
+    coinsPerAP = 7.89;
+
+    chestsPerOJ = 1/84;
+    chestsPerCider = 1/8.36;
+    chestsPerLemonade = 1/208.95;
+    chestsPerAP = 1/8.36;
+  }
```

Comment:
Good for now though a future version of this could do this live with data from the drop rate table :)